src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -183,6 +183,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('key_frame_interval_duration', help='ISO 8601 timespan duration of the key frame interval duration.')
         c.argument('access_token', help='The access token.')
         c.argument('description', help='The live event description.')
+        c.argument('ips', nargs='+', help='The IP allow list.')
+        c.argument('preview_locator', help='The preview locator Guid.')
+        c.argument('streaming_policy_name', help='The name of streaming policy used for live event preview.')
+        c.argument('alternative_media_id', help='An alternative media identifier associated with the preview URL. This identifier can be used to distinguish the preview of different live events for authorization purposes in the custom license acquisition URL template or the custom key acquisition URL template of the streaming policy specified in the streaming policy name field.')
+        c.argument('vanity_url', action='store_true', help='The live event vanity URL flag.')
+        c.argument('client_access_policy', help='The content of clientaccesspolicy.xml used by Silverlight.')
+        c.argument('cross_domain_policy', help='The content of crossdomain.xml used bySilverlight.')
+        c.argument('stream_options', nargs='+', help='The stream options.')
```

Comment:
Isn't it an enum type? Also, we should let the user know this parameter expects a space-separated list of ips (see other space-separated arguments)

Reply:
It's a list of string, that's why the options are not showing to the user.

Reply:
![image](https://user-images.githubusercontent.com/8594276/43745037-2fccd06a-99b3-11e8-8f59-fc2cced34857.png)

It seems it's a list of StreamOptionsFlag (LowLatency/Default)

Reply:
If you look at the parameter definitions list at the beginning of the class it says that it can be str or StreamOptionsFlag, I chose str because it was easier and it works like that. I can change it if it's a problem
![imagen](https://user-images.githubusercontent.com/40773617/43745119-827dedd0-99b3-11e8-8bae-a48e0945f638.png)